Include additions and bugfixes from @EpicKris

dirk-qualisys opened the pull request.

SmolSoftBoi committed:
Update CSS

SmolSoftBoi committed:
Create contribute

SmolSoftBoi committed:
Update Read me

SmolSoftBoi committed:
Add issue template

SmolSoftBoi committed:
Update

SmolSoftBoi committed:
Fix #7

SmolSoftBoi committed:
Fixed #6

SmolSoftBoi committed:
Fixed #4

SmolSoftBoi committed:
Add symbol

SmolSoftBoi committed:
Lint

SmolSoftBoi committed:
Fix classification cells

SmolSoftBoi committed:
Add first aid inhalation

SmolSoftBoi committed:
Add 5.1 media not to be used

SmolSoftBoi committed:
Fixed #4

SmolSoftBoi committed:
Fixed #8

SmolSoftBoi committed:
Fixed substance name

SmolSoftBoi committed:
Fixed exact value

SmolSoftBoi committed:
Add section 15.2.

dependabot[bot] committed:
Bump nanoid from 3.1.23 to 3.2.0

Bumps [nanoid](https://github.com/ai/nanoid) from 3.1.23 to 3.2.0.
- [Release notes](https://github.com/ai/nanoid/releases)
- [Changelog](https://github.com/ai/nanoid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ai/nanoid/compare/3.1.23...3.2.0)

---
updated-dependencies:
- dependency-name: nanoid
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

SmolSoftBoi committed:
Merge pull request #9 from SmolSoftBoi/dependabot/npm_and_yarn/nanoid-3.2.0

Bump nanoid from 3.1.23 to 3.2.0

SmolSoftBoi committed:
Bump deps

dependabot[bot] committed:
Bump minimist from 1.2.5 to 1.2.6

Bumps [minimist](https://github.com/substack/minimist) from 1.2.5 to 1.2.6.
- [Release notes](https://github.com/substack/minimist/releases)
- [Commits](https://github.com/substack/minimist/compare/1.2.5...1.2.6)

---
updated-dependencies:
- dependency-name: minimist
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

SmolSoftBoi committed:
Merge pull request #10 from SmolSoftBoi/dependabot/npm_and_yarn/minimist-1.2.6

Bump minimist from 1.2.5 to 1.2.6

dependabot[bot] committed:
Bump ansi-regex from 3.0.0 to 3.0.1

Bumps [ansi-regex](https://github.com/chalk/ansi-regex) from 3.0.0 to 3.0.1.
- [Release notes](https://github.com/chalk/ansi-regex/releases)
- [Commits](https://github.com/chalk/ansi-regex/compare/v3.0.0...v3.0.1)

---
updated-dependencies:
- dependency-name: ansi-regex
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

SmolSoftBoi committed:
Merge pull request #11 from SmolSoftBoi/dependabot/npm_and_yarn/ansi-regex-3.0.1

Bump ansi-regex from 3.0.0 to 3.0.1

SmolSoftBoi committed:
Bump deps

SmolSoftBoi committed:
Bump deps

dependabot[bot] committed:
Bump json5 from 2.2.1 to 2.2.3

Bumps [json5](https://github.com/json5/json5) from 2.2.1 to 2.2.3.
- [Release notes](https://github.com/json5/json5/releases)
- [Changelog](https://github.com/json5/json5/blob/main/CHANGELOG.md)
- [Commits](https://github.com/json5/json5/compare/v2.2.1...v2.2.3)

---
updated-dependencies:
- dependency-name: json5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

SmolSoftBoi committed:
Merge pull request #12 from SmolSoftBoi/dependabot/npm_and_yarn/json5-2.2.3

Bump json5 from 2.2.1 to 2.2.3

SmolSoftBoi committed:
Bump deps

SmolSoftBoi committed:
Create stale.yml

SmolSoftBoi committed:
Create node.js.yml

SmolSoftBoi committed:
Update Bootstrap version and stylelint
configuration

stacksharebot committed:
Create techstack.yml

stacksharebot committed:
Create techstack.md

stacksharebot committed:
Update techstack.yml

stacksharebot committed:
Update techstack.md

SmolSoftBoi committed:
Implement feature X to enhance user experience and optimize performance

SmolSoftBoi committed:
Merge pull request #16 from SmolSoftBoi/tech-stack-file

Create tech stack docs (techstack.yml and techstack.md)

SmolSoftBoi committed:
Update repository references to reflect correct ownership and maintain consistency

SmolSoftBoi commented:
@dirk-qualisys Are there plans to maintain this repo, as I am looking into updating mine to version 5.

google-labs-jules[bot] committed:
🧹 fix typo in comment: 'Mobility in soilMobility in soil'

Removed the repeated phrase 'Mobility in soil' in a comment within SDSComXSL.xsl to improve code readability.

Co-authored-by: SmolSoftBoi <491681+SmolSoftBoi@users.noreply.github.com>

google-labs-jules[bot] committed:
🧹 Remove commented out Bootstrap imports in scss/starter.scss

Commented out code is dead code that adds clutter without providing value.
Removing it improves readability and maintainability of the codebase.

Co-authored-by: SmolSoftBoi <491681+SmolSoftBoi@users.noreply.github.com>

SmolSoftBoi committed:
Merge pull request #23 from SmolSoftBoi/fix-comment-typo-mobility-in-soil-2613348891841498546

🧹 fix typo in comment: 'Mobility in soilMobility in soil'

google-labs-jules[bot] committed:
🧹 Remove commented out Bootstrap imports and fix CI cache

- Removed commented out Bootstrap imports in scss/starter.scss.
- Updated GitHub Actions workflow to use actions v4 and yarn cache.

Co-authored-by: SmolSoftBoi <491681+SmolSoftBoi@users.noreply.github.com>

google-labs-jules[bot] committed:
🧹 fix typo in comment and resolve CI issues

- Removed the repeated phrase 'Mobility in soil' in a comment within SDSComXSL.xsl.
- Updated GitHub Actions to v4 and fixed yarn cache configuration to resolve CI failures.

Co-authored-by: SmolSoftBoi <491681+SmolSoftBoi@users.noreply.github.com>

SmolSoftBoi committed:
Merge pull request #26 from SmolSoftBoi/fix-comment-typo-mobility-in-soil-2613348891841498546

🧹 fix typo in comment and resolve CI issues

SmolSoftBoi committed:
Merge pull request #25 from SmolSoftBoi/fix-commented-out-imports-3917339667710259837

🧹 Remove commented out Bootstrap imports in scss/starter.scss
